lava_reports/lava_reports/report/forecasted_transactions_report/forecasted_transactions_report.py

```diff
@@ -35,6 +35,10 @@ def execute(filters=None):
     if filters['filter_include_submitted'] == 'No':
         sql += F""" AND gl.name NOT LIKE %(status_id_key)s """
 
+    if filters['filter_accounts']:
```

Comment:
in this case filters_accounts is not mandatory, so it is possible that this if condition fail due to that the filters dict has no filters_accounts attribute